src/sentry.js

```diff
@@ -153,10 +154,22 @@ export const initializeSentry = () => {
 
     Sentry.init({
       dsn: key,
+
       ignoreErrors: [
         // RN's fetch implementation can raise these; we sometimes mimic it
         'Network request failed',
       ],
+
+      ...(Platform.OS === 'android'
+        ? {
+            // Disable Sentry's native code (libsentry.so), because it has
+            // memory-corruption bugs that cause the app to crash.
+            //   https://github.com/zulip/zulip-mobile/issues/5766
+            //   https://github.com/getsentry/sentry-java/issues/2955#issuecomment-1739030872
+            enableNdk: false,
```

Comment:
Greg found in testing that this, plus re-enabling Sentry on Android
    (coming up) seems adequate for #5766.

In this commit message let's include a link for that testing, since that's easy to do:
https://chat.zulip.org/#narrow/stream/243-mobile-team/topic/re-enabling.20Sentry/near/1654140